### PR TITLE
Timeline add new entry

### LINKED ISSUE #90: Add new entry
## Feature Description
- [ ] We should be able to click on the add new entry button on the timeline card and it should be able to create a new entry for a specific employer.
- [ ] You have the option to select what kind of entry it is and we should have a textbox to add this entry. (See screenshots)
- [ ] When you add a new entry, it should update the currently displayed entries: #91  

It should match the[ figma design](https://www.figma.com/file/ckkCZEVkae0kIXUwaQPbib/High-Fidelity-Designs?node-id=122%3A7840&t=4hEfLEf2WoCacOy2-0). Talk to designers for clarifications

## Implementation Suggestions
These are optional, if you have better ideas it's up to you to decide

- The entire component can rely on props that you can input as a parameter to tell it which employer data to work with, this is so that it's not completely dependent on the Employer page.
- When updating currently displayed entries, update UI data and update database at the same time. 

## Screenshots

Screen Shot 2023-02-07 at 3 12 01 PM
Screen Shot 2023-02-07 at 3 12 08 PM
Screen Shot 2023-02-07 at 3 12 14 PM

### PR DESCRIPTION
**Pull request by ganeshasapu**
# Description
Creates ability to add new notes to timeline for a given employer.

Possible Concerns:
- Relies on the employer page making fetch calls and formatting information before passing into timeline component
- Exact display and formatting (mostly for default look) should be corrected and polished when merging with #91.

Fixes #90 

Display:
Screen Shot 2023-02-28 at 2 22 26 PM

Example implementation in employer page:
Screen Shot 2023-02-28 at 2 20 36 PM

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have run `npm run lint:fix`
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by jordanjanakievski — CHANGES_REQUESTED** [review]
Clinet and Job Postings not displaying correctly after click

Screenshot 2023-03-02 at 21 43 14

Everything else is working and looking great

File: client/src/App.tsx [hunk 1]
```diff
@@ -2,25 +2,34 @@ import { useEffect, useState } from "react";
...
-				{Communication()}
-				<img src={logo} className="App-logo" alt="logo" />
-				<p>{!data ? "Loading..." : data}</p>
-			</header>
-		</div>
-	);
+  return (
+    <div className="App">
+      <header className="App-header">
+        {Communication()}
```
Comment (edwardhuahan) [inline client/src/App.tsx:19]: I would suggest to replace this with `<Communication />`

**Review by edwardhuahan — CHANGES_REQUESTED** [review]
Everything looks great! Just make sure to change that one line of code and it should be good

**Review by jordanjanakievski — APPROVED** [review]
Looks good to me

**Comment by edwardhuahan** [comment]
Looks great! good style

**Review by edwardhuahan — APPROVED** [review]

**Review by thardchi — APPROVED** [review]
Great job, everything looks good on my end!

**Pull request merged**